tests/generator-Tests/Unit-Tests/CodeGeneratorExpectedResults/JavaInterop1/WriteClass.txt

```diff
@@ -162,6 +162,7 @@ public partial class MyClass  {
 				_members.InstanceMethods.InvokeVirtualVoidMethod (__id, this, __args);
 			} finally {
 				JNIEnv.DeleteLocalRef (native_value);
+				global::System.GC.KeepAlive (value);
```

Comment:
***Does not need to be addressed for d16-8***: We don't need a `GC.KeepAlive()` for types which will be "deep marshaled".  This includes `System.String` and array types.  ***For master***, would we be able to *skip* this?